src/InfluxDB/Database.php

```diff
@@ -30,14 +33,40 @@ class Database
     protected $client;
 
     /**
-     * Precision constants
+     * @var InfluxDBV2Precision|InfluxDBV1Precision
      */
-    const PRECISION_NANOSECONDS = 'n';
-    const PRECISION_MICROSECONDS = 'u';
-    const PRECISION_MILLISECONDS = 'ms';
-    const PRECISION_SECONDS = 's';
-    const PRECISION_MINUTES = 'm';
-    const PRECISION_HOURS = 'h';
+    protected $precision;
```

Comment:
This should probably match what we expect as input from the constructor.
Either `InfluxDBPrecision` or `InfluxDBV1Precision|InfluxDBV2Precision`